src/main/java/org/plumelib/bcelutil/StackMapUtils.java

```diff
@@ -101,6 +106,21 @@
    */
   protected @NonNegative int number_active_locals;
 
+  /** The start of a local variable's live range: the first instruction in the range. */
+  protected InstructionHandle live_range_start = null;
+
+  /** The end of a local variable's live range: the first instruction after the range. */
```

Comment:
Is this comment correct?  I added the final clause.

Reply:
The JVM definition is [start_pc, start_pc + length) but BCEL uses [start_pc, end_pc).  Your comment is correct.  There is some nasty bits dealing with a live range that ends before the last instruction in the method or after the last instruction in the method, but we don't need to go into that detail here.